src/main/resources/db/migration/all/20240513133000__create_cas_2_summary_view.sql

```diff
@@ -0,0 +1,21 @@
+-- create application summary view
+DROP VIEW IF EXISTS cas_2_application_summary;
```

Comment:
could the live table creating be put into the same sql file? They seems to depend on each other, and as they haven't yet been merged into main we are free to move them about

Reply:
Yes good idea!  Haver merged the two sql files into one.